client/http.go

```diff
@@ -228,7 +228,7 @@ func (c *HTTPClient) MLSelectTests(ctx context.Context, stepID, mlKey, source, t
 	if err := c.validateMLSelectTestArgs(); err != nil {
 		return resp, err
 	}
-	path := fmt.Sprintf(mlSelectTestsEndpoint, c.AccountID, c.OrgID, c.ProjectID, c.PipelineID, c.BuildID, c.StageID, stepID, c.Repo, c.Sha, source, target, mlKey)
+	path := fmt.Sprintf(mlSelectTestsEndpoint, c.AccountID, c.OrgID, c.ProjectID, c.PipelineID, c.BuildID, c.StageID, stepID, c.Repo, c.Sha, source, target, mlKey, c.CommitLink)
```

Comment:
Do you need a non empty check on commit link?

Reply:
Nope, its fine if its not present as well